lib/readGltfGeometry.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+var readAccessor = require('./readAccessor');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var modeToPrimitiveType = require('./modeToPrimitiveType');
+var Cesium = require('cesium');
+var Geometry = Cesium.Geometry;
+var GeometryAttribute = Cesium.GeometryAttribute;
+var ComponentDatatype = Cesium.ComponentDatatype;
+
+module.exports = readGltfGeometry;
+
+function readGltfGeometry(gltf, primitive) {
+    var positionId = primitive.attributes.POSITION;
+    var positionAccessor = gltf.accessors[positionId];
+    var componentsPerAttribute = numberOfComponentsForType(positionAccessor.type);
+    var values = readAccessor(gltf, positionAccessor);
+    
+    var packedValues = [];
+    for (var i = 0; i < values.data.length; i++) {
+        packedValues.push(values.data[i].x);
+        packedValues.push(values.data[i].y);
+        packedValues.push(values.data[i].z);
+    }
```

Comment:
`Cartesian3.packArray` does not seem to be exposed by Cesium. However, I will change it to at least use `Cartesian3.pack` in the loop instead, for the time being.